API.markdown

```diff
@@ -107,10 +113,84 @@ With the default id left as is (i.e. "g3log") a creation of the logger in the un
 ```
 The resulting filename would be something like: 
 ```
-./(ReplaceLogFile).g3log.20160217-001406.log
+   ./(ReplaceLogFile).g3log.20160217-001406.log
 ```
 
 
+## Custom LOG <a name="log_formatting">formatting</a>
+### Overriding the Default File Sink's file header
+The default file header can be customized in the default file sink in calling 
+```
+   FileSink::overrideLogHeader(std::string);
+```
+
+
+### Overriding the Default FileSink's log formatting
+The default log formatting is defined in `LogMessage.hpp`
+```
+   static std::string DefaultLogDetailsToString(const LogMessage& msg);
+```
+
+### Adding thread ID to the log formatting
+An "all details" log formatting function is also defined - this one also adds the "calling thread's ID"
+```
+   static std::string DefaultLogDetailsToString(const LogMessage& msg);
```

Comment:
Believe this should be `FullLogDetailsToString(...)`

Reply:
Thanks. I’ll go over the doc changes and clear things up